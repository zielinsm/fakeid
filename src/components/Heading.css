.heading {
  height: 250px;
  border-radius: unset !important;
}

.heading__wrapper {
  height: 100%;	
}

.heading__slogan {
  font-size: 2em !important;	
}

.heading__description {
  font-size: 1.5em !important;	
  margin-top: -0.5em !important;
}