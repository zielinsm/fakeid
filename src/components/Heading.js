import React from "react";
import { Segment, Grid, Header } from "semantic-ui-react";
import "./Heading.css";

const Heading = () => (
  <Segment inverted className="heading">
    <Grid centered verticalAlign="middle" className="heading__wrapper">
      <Grid.Column>
        <Header
          as="h1"
          content="FakeID"
          textAlign="center"
          inverted
          className="heading__slogan"
        />
        <Header
          as="h2"
          content="Generate fake identites easily."
          textAlign="center"
          inverted
          className="heading__description"
        />
      </Grid.Column>
    </Grid>
  </Segment>
);

export default Heading;
