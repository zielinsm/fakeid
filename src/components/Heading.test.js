import React from "react";
import Heading from "./Heading";
import renderer from "react-test-renderer";

test("Heading renders correctly", () => {
  const component = renderer.create(<Heading />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
