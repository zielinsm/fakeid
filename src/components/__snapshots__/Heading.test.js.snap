// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
<div
  className="ui inverted segment heading"
>
  <div
    className="ui centered middle aligned grid heading__wrapper"
  >
    <div
      className="column"
    >
      <h1
        className="ui inverted center aligned header heading__slogan"
      >
        FakeID
      </h1>
      <h2
        className="ui inverted center aligned header heading__description"
      >
        Generate fake identites easily.
      </h2>
    </div>
  </div>
</div>
`;
